EpsilonLights/Inc/Lights.h

```diff
@@ -31,7 +31,7 @@
 #define BRAKES_INPUT_INDEX_P1 3 // 24 = [*3*] >> 0
 #define BRAKES_INPUT_INDEX_P2 0 // 24 = [3] >> *0*
 
-#define BATTERY_STAT_STDID 0x6FBU
+#define BATTERY_STAT_ERRORS_STDID 0x303
```

Comment:
Can you explain this change?

Reply:
The new Orion BMS has the errors on ID 0x303

Reply:
I did forget to add in U though